src/con_duct/__main__.py

```diff
@@ -330,7 +328,9 @@ def collect_sample(self) -> Sample:
             )
             for line in output.splitlines()[1:]:
                 if line:
-                    pid, pcpu, pmem, rss_kib, vsz_kib, etime, cmd = line.split(maxsplit=6)
+                    pid, pcpu, pmem, rss_kib, vsz_kib, etime, cmd = line.split(
+                        maxsplit=6
+                    )
```

Comment:
these changes are unrelated but the result of not having pre-commit "install"ed/used to `black` the code.